### PR TITLE
Change Tensorflow Serial Test Part 2 and bigdl part 3

### PR DESCRIPTION
**Pull request by Quincy2014**
## What changes were proposed in this pull request?

Change tensorflow serial test from ModuleSerializerSpec to each layer's unit test.
Change four bigdl layers serial test from ModuleSerializerSpec to each layer's unit test.

## How was this patch tested?

Jenkins.

### CODE REVIEW

**Comment by Quincy2014** [comment]
Jenkins passed. http://172.168.2.101:8080/job/BigDL-PR-Validation-v3/299/

**Comment by Quincy2014** [comment]
Jenkins passed. http://172.168.2.101:8080/job/BigDL-PR-Validation-v3/316/

File: spark/dl/src/test/scala/com/intel/analytics/bigdl/torch/MaxoutSpec.scala [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/*
```
Comment (wzhongyuan) [inline spark/dl/src/test/scala/com/intel/analytics/bigdl/torch/MaxoutSpec.scala]: it should not be move to this package, keep the original one and updated the package info

**Comment by Quincy2014** [comment]
Jenkins passed. http://172.168.2.101:8080/job/BigDL-PR-Validation-v3/319/

**Comment by wzhongyuan** [comment]
LGTM

**Pull request merged**